### DISCUSSION TITLE
Default rules for AKS NetworkPolicy

### DISCUSSION THREAD

**Discussion by Gordonby** [Azure/PSRule.Rules.Azure#1043] [Ideas]
My template is failing with;
`
| REASON:
    | - The field 'Properties.networkProfile.networkPolicy' is set to 'calico'.
`

I'd suggest that either azure or calico should be valid for satisfying the recommendation.
`
RECOMMEND:
    | Azure Network Policies improve cluster and workload security by limiting
    | network communication.
    | Network Policy is a deployment time configuration. Consider redeploying the AKS
    | cluster with Network Policy enabled.
`

ref: https://docs.microsoft.com/en-us/azure/aks/use-network-policies#network-policy-options-in-aks

**Comment by BernieWhite** [comment]
@Gordonby Yes I agree. Calico meets the WAF intent of this rule. Logged issue #1046
